java/core/src/test/java/co/worklytics/psoxy/rules/RulesUtilsTest.java

```diff
@@ -47,7 +48,8 @@ public void setup() {
         container.inject(this);
     }
 
-    static final String BASE64_YAML_REST = "YWxsb3dBbGxFbmRwb2ludHM6IGZhbHNlCmVuZHBvaW50czoKICAtIHBhdGhSZWdleDogL2NhbGVuZGFyL3YzL2NhbGVuZGFycy8uKi9ldmVudHMuKgogICAgdHJhbnNmb3JtczoKICAgICAgLSAhPHBzZXVkb255bWl6ZT4KICAgICAgICBqc29uUGF0aHM6CiAgICAgICAgICAtICQuLmVtYWlsCiAgICAgIC0gITxyZWRhY3Q+CiAgICAgICAganNvblBhdGhzOgogICAgICAgICAgLSAkLi5kaXNwbGF5TmFtZQogICAgICAgICAgLSAkLml0ZW1zWypdLmV4dGVuZGVkUHJvcGVydGllcy5wcml2YXRlCg==";
+    static final String BASE64_YAML_REST = "YWxsb3dBbGxFbmRwb2ludHM6IGZhbHNlCmVuZHBvaW50czoKICAtIHBhdGhSZWdleDogIl4vKHYxLjB8YmV0YSkvdXNlcnMvP1teL10qIgogICAgdHJhbnNmb3JtczoKICAgIC0gITxyZWRhY3Q+CiAgICAgIGpzb25QYXRoczoKICAgICAgLSAiJC4uZGlzcGxheU5hbWUiCiAgICAgIC0gIiQuLmVtcGxveWVlSWQiCg==";
+    static final String BASE64_YAML_REST_COMPRESSED = "H4sIAAAAAAAAAFWMPQvCQBBE+/yK9bBQwZy2QRQLCxsRWzGwepsY2dwdt+dHwB9vEkSwnJn3Bpndc828sca7ykbJoEAWSuhXJABT8BivByrplYHK9egxT2fvM0Uc67tQEL065vo0US0LEANaKVyoe7ezB4tABi9x2WeAmzi7bx+/QIeoYZqaSjxjs8Oa1N9AtWfXEG2NSj7mG4K5sgAAAA==";
```

Comment:
Just curious; 241 to 217 characters compressed, less than 10%

Reply:
yeah, but this is a small example. I would expect real-life large rule sets to gain proportionally more